### PR TITLE
Player overlays now centered

### PR DESCRIPTION
**Pull request by avently**
#### What is it?
- [x] Bug fix (user facing)

#### Description of the changes in your PR
Overlays were off-centered before this PR. By overlays I mean volume slider, brightness slider, loading icon, etc. Now it's fixed.
Also excluded play/prev/next buttons from swipe interception so now it's not possible to drag them up/down with expectation to drag the whole player.
If you'll find any issue, please, fix it yourself or just drop the PR because waiting of the fix from my side will take months. Thank you for understanding.

P.s. so many changes in player.xml in diff view but actually it's just the overlays were included into top level relativelayout to be able to position them via one assigment of paddings. For review it's easier to open previous player.xml and new player.xml side-by-side.

#### Fixes the following issue(s)
https://github.com/TeamNewPipe/NewPipe/issues/4332#issuecomment-701459044
 

#### Testing apk
[overlays.zip](https://github.com/TeamNewPipe/NewPipe/files/5308821/overlays.zip)

#### Agreement
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

**Comment by opusforlife2** [comment]
Confirmed! Thanks a lot!

So instead of those 3 blocks of code you made a `for` loop.

**Review by Stypox — APPROVED** [review]
The code looks good to me, thank you :-D

**Review by TobiGr — APPROVED** [review]
Thank you @avently. You are an absolute machine!

**Review by TobiGr — APPROVED** [review]

**Comment by opusforlife2** [comment]
You approved these changes after merging as well? You reeeeeally like the code, it seems. :P

**Pull request merged**
